### PR TITLE
Clean up and update cards for WWZTo4L_4f_NLO for UL

### PR DESCRIPTION
**Pull request by singh-ramanpreet**
Removed one directory and renamed `WWZTo4L_new_4f_NLO` to `WWZTo4L_4f_NLO`.

Can you also please review the cards for UL production?

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WWZJets/WWZTo4L_4f_NLO/WWZTo4L_4f_NLO_proc_card.dat [hunk 1]
```diff
@@ -5,5 +5,5 @@ define lep- = e- mu- ta-
 # exclude s-channel top contribution to avoid
 # overlap with TTZ sample
 generate p p > w w lep+ lep- $$ t t~ h [QCD]
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WWZJets/WWZTo4L_4f_NLO/WWZTo4L_4f_NLO_proc_card.dat:7]: What is $$ t t~ , How can it get such contributions

Comment (singh-ramanpreet) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WWZJets/WWZTo4L_4f_NLO/WWZTo4L_4f_NLO_proc_card.dat:7]: I assume, that's what original requester intended. To exclude `t t~ h` s-channel.

> How can it get such contributions

@Saptaparna do you know about this?

**Comment by Saptaparna** [comment]
Yes, indeed. We always remove any possible contribution from top. But you are right, I don't think this process really acquires top contributions. I can run MG5 at LO and produce the diagrams and double check.

**Comment by sihyunjeon** [comment]
@Saptaparna I checked and  generate p p > w w lep+ lep- $$ t t~ h [QCD]   generate p p > w w lep+ lep- $$ h [QCD]  had no difference in number of diagrams

**Comment by Saptaparna** [comment]
Great! Thanks @sihyunjeon! Unless @agrohsje has any further comments, the cards look fine and can be used for production?

**Comment by sihyunjeon** [comment]
@Saptaparna @singh-ramanpreet can we remove the NULL commands ($$ t t~)? I think that was the policy which Alexander was hoping for. Keeping meaningful lines only

**Comment by Saptaparna** [comment]
Yes, let's do that! @singh-ramanpreet do you agree? The run card is fine, right?

**Comment by singh-ramanpreet** [comment]
@Saptaparna 

If the number of diagrams are same, then I will remove the redundant part.
Only thing that I want to know, why it was there in first place? My guess is, it's maybe required in older version of madgraph.

**Comment by singh-ramanpreet** [comment]
removed `$$ t t~`

**Comment by singh-ramanpreet** [comment]
@Saptaparna @sihyunjeon @agrohsje is it all good? then we can start working producing gridpack.

**Comment by sihyunjeon** [comment]
looks good to me

**Comment by Saptaparna** [comment]
looks good to me too 👍! @agrohsje: any further comments?

**Comment by agrohsje** [comment]
removed the comment about s-channel. going to merge now.

**Pull request merged**